broker-daemon/utils/grpc-gateway.js

```diff
@@ -71,7 +80,10 @@ const middleware = (protoFiles, grpcLocation, credentials = grpc.credentials.cre
         s.methods.forEach(m => {
           if (m.options[GRPC_API_OPTION_ID]) {
             supportedMethods.forEach(httpMethod => {
-              if (m.options[GRPC_API_OPTION_ID][httpMethod]) {
+              // We should limit methods to those included on the whitelist
+              const shouldGateway = whitelist.includes(WILDCARD) || whitelist.includes(m.options[GRPC_API_OPTION_ID][httpMethod])
```

Comment:
[nit] `shouldGateway` is a tough name because we are in the gateway... unfortunately I dont have suggestions on how we can make variable describe the action better. Maybe something like `allowMethod` or `permitMethod`